VideoLocker/res/layout/row_discussion_user_profile.xml

```diff
@@ -13,44 +13,37 @@
         android:id="@+id/profile_image"
         android:layout_width="@dimen/discussion_image_diameter"
         android:layout_height="@dimen/discussion_image_diameter"
+        android:layout_gravity="center_vertical"
         android:layout_marginEnd="@dimen/discussion_responses_standard_margin"
         android:layout_marginRight="@dimen/discussion_responses_standard_margin"
         android:src="@drawable/profile_photo_placeholder" />
 
     <LinearLayout
         android:layout_width="wrap_content"
-        android:layout_height="match_parent"
+        android:layout_height="wrap_content"
         android:gravity="center_vertical"
         android:orientation="vertical">
 
         <TextView
             android:id="@+id/discussion_author_text_view"
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:textColor="@color/edx_grayscale_neutral_base"
-            android:textSize="@dimen/edx_xxx_small"
+            style="@style/discussion_regular_text"
+            android:textDirection="locale"
```

Comment:
Since the layout width is defined as `wrap_content`, the text direction makes no difference in this case. I suppose it can be defined out of principle, but in that case it would probably be preferable to do so globally in the default `TextView` style instead of in each individual `TextView`.